serverless/cmd/sequence/main.go

```diff
@@ -39,13 +39,27 @@ var (
 
 func main() {
 	flag.Parse()
+	var err error
+
+	toAdd, err := filepath.Glob(*entries)
+	if err != nil {
+		glog.Exitf("Failed to glob entries %q: %q", *entries, err)
+	}
+	if toAdd == nil {
```

Comment:
If we're dealing with slices, then it's better form to always use `len(thing)` for these checks - it'll happily return `0` for zero sized slices, as well as `nil` slices.